tests/tdklambdagenesys.py

```diff
@@ -0,0 +1,82 @@
+'''
+Tests for the TDK Genesys PSU
+@author Eilidh Southren
+'''
+
+import unittest
+from utils.channel_access import ChannelAccess
+from utils.testing import get_running_lewis_and_ioc
+
+# MACROS to use for the IOC
+# Macros may need to be formally set - check dbLoadRecords line in *ioc*_ddevsim_ioc.log file
+MACROS = { "ADDR1": "1", "PORT1": "1"}
+
+class OutputMode(object):
+    VOLTAGE = "VOLTAGE"
+    CURRENT = "CURRENT"
+
+class TdklambdagenesysTests(unittest.TestCase):
+
+
```

Comment:
Only one blank line between stuff inside a class, for example, between methods.

There are more superfluous blank lines below, I won't highlight them all.